build.gradle

```diff
@@ -280,7 +310,7 @@ node {
     distBaseUrl = 'https://nodejs.org/dist'
     download = true
     npmVersion = '6.14.17'
-    version = '14.20.0'
+    version = '14.20.1'
```

Comment:
security patch, and also a test of whether the bamboo environment supports gradle downloading a private node/npm environment (it does)